<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SGEditorModal</class>
 <widget class="QDialog" name="SGEditorModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>59</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="v2">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>311</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>449</width>
      <height>309</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
